### PR TITLE
feat(remove-android): Update docs

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Details

Remove Android references from docs

##### Motivation

Android deprecation feature

#### Pull request checklist

- [n/a] Addresses an existing issue (part of Android deprecation feature)
- [x] Ran `yarn fastpass`
- [n/a] Added/updated relevant unit test(s) (and ran `yarn test`)
- [n/a] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,17 @@ Copyright (c) Microsoft Corporation. All rights reserved.
...
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/users/pbjjkligggfmakdaogkfomddhfmpjeni.svg)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/stars/pbjjkligggfmakdaogkfomddhfmpjeni.svg)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni/reviews)
 
-Two projects are built from this repository:
+One project is built from this repository:
 
 -   **Accessibility Insights for Web** is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
--   **Accessibility Insights for Android** is a cross-platform desktop tool used for testing accessibility of Android applications.
 
 ### Notice
```
Comment (DaveTryon) [inline README.md:18]: @codeofdusk, I left the deprecation notice in place. Would you prefer to update it now or in a separate PR?

File: README.md [hunk 2]
```diff
@@ -3,18 +3,17 @@ Copyright (c) Microsoft Corporation. All rights reserved.
...
 
 [![Build Status](https://dev.azure.com/accessibility-insights/accessibility-insights-web/_apis/build/status/accessibility-insights-web%20CI?branchName=main)](https://dev.azure.com/accessibility-insights/accessibility-insights-web/_build/latest?definitionId=37&branchName=main)
 [![codecov](https://codecov.io/gh/microsoft/accessibility-insights-web/branch/main/graph/badge.svg)](https://codecov.io/gh/microsoft/accessibility-insights-web)
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/v/pbjjkligggfmakdaogkfomddhfmpjeni.svg?label=Version)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/users/pbjjkligggfmakdaogkfomddhfmpjeni.svg)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/stars/pbjjkligggfmakdaogkfomddhfmpjeni.svg)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni/reviews)
 
-Two projects are built from this repository:
+One project is built from this repository:
```
Comment (codeofdusk) [inline README.md:14]: Maybe reword this section now that it's just a single project?

File: README.md (hunk 1 above)
Comment (codeofdusk) [reply README.md:18]: Maybe just remove it?

**Review by codeofdusk — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (DaveTryon) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (DaveTryon) [reply README.md]: Done
